### PR TITLE
[GLUTEN-2491][CH] Fix sequence with null value as argument

### PR DESCRIPTION
**Pull request by exmy**
## What changes were proposed in this pull request?

Fix sequence with null value as argument and move the convert implemention of sequence to function parser of ch backend.

(Fixes: \#2491)

## How was this patch tested?

UT

### CODE REVIEW

File: cpp-ch/local-engine/Parser/scalar_function_parser/sequence.cpp [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        const auto * result_node = toFunctionNode(actions_dag, "multiIf", {
+            start_is_null_node,
+            null_const_node,
+            end_is_null_node,
+            null_const_node,
+            step_is_null_node,
+            null_const_node,
+            modulo_step_eq_zero_node,
+            wrap_range_1_node,
+            wrap_range2_node
```
Comment (taiyang-li) [inline cpp-ch/local-engine/Parser/scalar_function_parser/sequence.cpp:116]: rename to wrap_range_2_node

**Review by taiyang-li — APPROVED** [review]

**Comment by exmy** [comment]
@rui-mo Could you help trigger the failed CI job? Thanks!

**Comment by rui-mo** [comment]
> @rui-mo Could you help trigger the failed CI job? Thanks!

@exmy Retriggered, thanks.

**Comment by exmy** [comment]
> [](/apps/github-actions) ** Velox backend / velox-ubuntu2004-test-spark33-slow (pull_request) **

There is another one `Velox backend / velox-ubuntu2004-test-spark33-slow (pull_request) ` also needs to be retriggered @rui-mo

**Review by liuneng1994 — APPROVED** [review]
LGTM

**Pull request merged**